cli/package.json

```diff
@@ -52,6 +52,7 @@
     "fs-extra": "^10.1.0",
     "global": "^4.4.0",
     "globby": "^13.1.3",
+    "golden-fleece": "^1.0.9",
```

Comment:
does this retain comments?
https://www.npmjs.com/package/golden-fleece

Reply:
Yes, exactly, that's why I picked that! `json5` cannot do this.

Reply:
that's awesome!! I'd been wanting a library that could do this!

I need to add it do by maintain-everything library, ember-apply.

Double thanks for this PR, and introducing me to a json5 library that is actually good :upside_down_face:

Reply:
I only found it by specifically looking for something that can preserve comments, as we have them in `tsconfig.json` (you might remember 🙃). It's made my Rich Harris, so that gave me some confidence it wouldn't be too bad. Although the last commit is 5 years old as I just saw... 🤔